ProgressBar/themes/bootstrap/variables.less

```diff
@@ -1,2 +1,3 @@
-@transition-params: 0.3s linear 0s;
-@line-height: 1.6em;
\ No newline at end of file
+@height: 1.8em; // Controls the height of the widget (180% of the font-size)
```

Comment:
We also have https://github.com/ibm-js/deliteful/blob/master/docs/Button.md#styling-css-classes
Feel free to create an issue.